files_opds/lib/meta.php

```diff
@@ -64,7 +64,7 @@ public static function isValid($meta) {
 	protected static function load($id) {
                 $sql = 'SELECT * FROM `*PREFIX*opds_metadata` WHERE id = ?';
                 $args = array($id);
-                $query = \OCP\DB::prepare($sql);
+                $query = \OC_DB::prepare($sql);
```

Comment:
Same for this. This is internal API now, that can change at any point in time. Best is to use the DB connection: https://github.com/nextcloud/server/pull/8985/files#diff-47cd1c326a2fae00c6f0e1e088c994bbL430 and there the query builder is currently the way to go: https://docs.nextcloud.com/server/stable/developer_manual/app/storage/database.html (there is one more abstraction layer in the docs: the mapper and entities - you don't need them, but inside them is basically all the DB logic that you need and then your code is better separated. You could use the DB access also without mapper and entity - it's just a best practice here)